src/index.js

```diff
@@ -83,7 +83,7 @@ class Service {
       return result[0];
     })
     .then(select(params, this.id))
-    .catch(utils.errorHandler);
+    .catch(() => { throw new errors.NotFound(`No record found for id '${id}'`); });
```

Comment:
That is correct. Since we are doing this in the get, in most cases it should be the correct behaviour. 
Again it might comes down to how you want to handle types. If a query parameter uses the incorrect type should an error be thrown? Or should there just be a 'Not Found' error? This catch still might be too broad regardless though - like what if the DB is not available.

In any case my thoughts were that there are probably better ways to do this, but since the tests are passing and it is quite important to get the tests running against a 'proper' DB it is probably a good idea to just run with this for now. (From my side it seems quite problematic using SQLLite for tests because it behaves differently to most other RDB's and in addition the code has some branches for specific dialects)